### DISCUSSION TITLE
Enhancement for paginator

### DISCUSSION THREAD

**Discussion by S-Bra** [iTwin/iTwinUI#2183] [Ideas]
The right edge of the paginator is used by the "items per page" control. The left edge is unused and it would be great if it allows to place custom control. 
We overlay the paginator and draw another select in the left corner to allow cycling through the pages.
![image](https://github.com/user-attachments/assets/329e2974-8756-4dde-b0c8-45354fc90ace)

**Comment by mayank99** [comment]
The left edge is not always unused. This is where we typically display the row selection count (see [example](https://itwin.github.io/iTwinUI/react/?story=table--with-paginator&theme=dark)). Does your Table not have selection enabled? From the screenshot, it does not even look like a Table.

**Reply by S-Bra** [reply]
The paginator is not used with a table. The pagination is needed to navigate though time-series-charts. 
![image](https://github.com/user-attachments/assets/4fef60ef-0c88-4a18-8d11-8b58c398d62b)

**Reply by mayank99** [reply]
Got it. Yeah, this is very much an unsupported use-case right now. I think it would make more sense for iTwinUI to provide a general `Paginator` that is intended to be used outside of `Table`. We can support customizing the left side in that general component.
